iocore/net/SSLUtils.cc

```diff
@@ -1474,9 +1476,9 @@ ssl_callback_info(const SSL *ssl, int where, int ret)
     if (cipher) {
       const char *cipherName = SSL_CIPHER_get_name(cipher);
       // lookup index of stat by name and incr count
-      InkHashTableValue data;
-      if (ink_hash_table_lookup(ssl_cipher_name_table, cipherName, &data)) {
-        SSL_INCREMENT_DYN_STAT((intptr_t)data);
+      auto found = ssl_cipher_name_table.find(cipherName);
+      if (found != ssl_cipher_name_table.end()) {
+        SSL_INCREMENT_DYN_STAT((intptr_t)found->second);
```

Comment:
You don't need this cast, it's already an `intptr_t`. I suspect you don't need the cast in line 1077 either.